File: src/main/java/me/kuku/yuq/service/QQService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.QQEntity;

import java.util.List;

@AutoBind
public interface QQService {
    QQEntity findByQQAndGroup(Long qq, Long group);
    void save(QQEntity qqEntity);
    @SuppressWarnings("UnusedReturnValue")
    int delByQQAndGroup(Long qq, Long group);
    List<QQEntity> findAll();
    List<QQEntity> findByHostLocPush(boolean hostLocPush);
}

File: src/main/kotlin/me/kuku/yuq/service/QQService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.QQEntity

@AutoBind
interface QQService {
    fun findByQQAndGroup(qq: Long, group: Long): QQEntity?
    fun save(QQEntity: QQEntity)
    fun delByQQAndGroup(qq: Long, group: Long): Int
    fun findAll(): List<QQEntity>
}